index.md

```diff
@@ -37,8 +37,8 @@ our meetings.
 Where:
 -----
 
-Data Science Initiative Classroom </br>
-Third Floor Shields Library, Rm 360 </br>
+Data Science Initiative Classroom       
+Third Floor Shields Library, Rm 360      
```

Comment:
Good question! Markdown has a weird syntax, 3+ white spaces trailing a line is equivalent to `\n` a new line character. You can see how the rendered page looks like at http://karenyyng.github.io/davis/. The new line is rendered correctly. :)